sections/04_study.md

```diff
@@ -27,8 +27,55 @@ dive into merits or weaknesses of individual methods.*
 
 ### Gene expression
 
-*Predicting gene expression levels and unsupervised approaches for learning
-from gene expression.  Those could be divided into separate sub-sections.*
+Gene expression measurements characterize the abundance of many thousands of
+RNA transcripts within a given organism, tissue, or cell. This characterization
+can represent the underlying state of the given system and can be used to study
+heterogeneity across samples as well as how the system reacts to perturbation.
+While gene expression measurements have been traditionally made by quantitative
+PCR, low throughput flourescense based methods, and microarray technologies, the
+field has shifted in recent years to primarily performing RNA sequencing to
+catalog total RNA. As such next generation sequencing technologies continue to
+fall in price and rise in throughput, studying gene expression through the lens
+of deep learning will increase in practicality and promise. 
+
+Already several deep learning approaches have been applied to gene expression
+data with varying aims. For instance, many researchers have applied unsupervised
+deep learning approaches to extract meaningful representations of gene modules
+or sample clusters. Unsupervised approaches, such as denoising autoencoders,
+have been applied to cluster yeast expression microarrays into known modules
+representing cell cycle processes [@tag:Gupta2015_exprs_yeast] and also to
+stratify yeast strains based on chemical and mutational perturbations
+[@tag:Chen2016_exprs_yeast]. Shallow (one hidden layer) denoising autoencoders
+have also been fruitful in extracting biological insight from thousandas of
+_Pseudomonas aeruginosa_ experiments [@tag:Tan2015_adage @tag:Tan2016_eadage]
+and in aggregating features relevant to specific breast cancer subtypes
+[@tag:Tan2014_psb]. Alternatively, deep learning approaches are being applied
```

Comment:
I'd like to see another line describing why these methods have been fruitful, e.g., what was learned.  This may be a good place to mention advantages of the unsupervised approach.  If you add a line or two here, "Alternatively" could start a new paragraph.